common/djangoapps/track/views/__init__.py

```diff
@@ -81,7 +81,7 @@ def user_track(request):
         except ValueError:
             pass
 
-    context_override = contexts.course_context_from_url(page)
+    context_override = {}
```

Comment:
Actually we don't want to change the call in `user_track` since the URL on the HTTP request is always the tracking endpoint URL, so the actual URL of the calling page is passed in through the `page` parameter.

Reply:
Got it :+1: